_strategies/prototyping.md

```diff
@@ -24,37 +24,35 @@ The following principles are meant to guide product teams in the use of technica
 
 ### Avoid Premature Optimization
 
-An acquisition prototype should support a procurement, not become an impediment to it. It should be developed rapidly, using a relatively lightweight approach and at low cost. Guard against over-optimization.
+A prototype should support a procurement and be thought of as a research tool. It should be developed rapidly, using a relatively lightweight approach and at low cost. Guard against over-optimization.
 
-Unlike other kinds of prototypes, acquisition prototypes are not typically used as a way to obtain user feedback, but rather to understand the mechanics behind a hypothetical user's experience. It should approximate a user’s experience to help identify risk, and to help validate technical decisions.
+The government should leverage prototypes to garner obtain user feedback, validate technical decisions, and better understand the organizational barriers to deploying software in modern ways.
```

Comment:
Does `garner` or `obtain` make the most sense here?

Reply:
ah, yes, will update.